main.tf

```diff
@@ -0,0 +1,36 @@
+data "aws_caller_identity" "default" {}
+
+module "label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.5.3"
+  enabled    = "${var.enabled}"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  name       = "${var.name}"
+  delimiter  = "${var.delimiter}"
+  attributes = "${var.attributes}"
+  tags       = "${var.tags}"
+}
+
+resource "aws_iam_account_alias" "default" {
+  count = "${var.enabled == "true" ? 1 : 0}"
+
+  account_alias = "${module.label.id}"
+}
+
+resource "aws_iam_account_password_policy" "this" {
+  count = "${var.password_policy_enabled == "true" ? 1 : 0}"
```

Comment:
let's add `var.enabled == "true" as well since it's a flag to enable/disable the entire module

```hcl
count = "${var.enabled == "true" && var.password_policy_enabled == "true" ? 1 : 0}"
```

Reply:
I struggled with that b/c it felt like they weren't mutually exclusive. Thoughts?